packages/logo/src/logos/MonochromeLogoMark.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { SVGLogoProps } from '../types';
+
+export default function MonochromeLogoMark({ fill, className }: SVGLogoProps) {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
```

Comment:
I don't think SVGs need this anymore, but could be wrong.

Reply:
👍